Makefile

```diff
@@ -423,84 +349,16 @@ server-lint: ## Run linters on server code.
-	@echo Starting docker container for mysql
-	docker-compose -f ./docker-testing/docker-compose-mysql.yml down -v --remove-orphans
-	docker-compose -f ./docker-testing/docker-compose-mysql.yml run start_dependencies
-	cd server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=server-mysql-profile.coverage -count=1 -timeout=30m ./...
-	cd server; go tool cover -func server-mysql-profile.coverage
-	cd mattermost-plugin/server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=plugin-mysql-profile.coverage -count=1 -timeout=30m ./...
-	cd mattermost-plugin/server; go tool cover -func plugin-mysql-profile.coverage
-	docker-compose -f ./docker-testing/docker-compose-mysql.yml down -v --remove-orphans
-
-server-test-mariadb: export FOCALBOARD_UNIT_TESTING=1
-server-test-mariadb: export FOCALBOARD_STORE_TEST_DB_TYPE=mariadb
-server-test-mariadb: export FOCALBOARD_STORE_TEST_DOCKER_PORT=44445
-
-server-test-mariadb: templates-archive ## Run server tests using mysql
-	@echo Starting docker container for mariadb
-	docker-compose -f ./docker-testing/docker-compose-mariadb.yml down -v --remove-orphans
-	docker-compose -f ./docker-testing/docker-compose-mariadb.yml run start_dependencies
-	cd server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=server-mariadb-profile.coverage -count=1 -timeout=30m ./...
-	cd server; go tool cover -func server-mariadb-profile.coverage
-	cd mattermost-plugin/server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=plugin-mariadb-profile.coverage -count=1 -timeout=30m ./...
-	cd mattermost-plugin/server; go tool cover -func plugin-mariadb-profile.coverage
-	docker-compose -f ./docker-testing/docker-compose-mariadb.yml down -v --remove-orphans
-
-server-test-postgres: export FOCALBOARD_UNIT_TESTING=1
-server-test-postgres: export FOCALBOARD_STORE_TEST_DB_TYPE=postgres
-server-test-postgres: export FOCALBOARD_STORE_TEST_DOCKER_PORT=44447
-
-server-test-postgres: ## Run server tests using postgres
-	@echo Starting docker container for postgres
-	docker-compose -f ./docker-testing/docker-compose-postgres.yml down -v --remove-orphans
-	docker-compose -f ./docker-testing/docker-compose-postgres.yml run start_dependencies
-	cd server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=server-postgres-profile.coverage -count=1 -timeout=30m ./...
-	cd server; go tool cover -func server-postgres-profile.coverage
-	cd mattermost-plugin/server; go test -tags '$(BUILD_TAGS)' -race -v -coverpkg=./... -coverprofile=plugin-postgres-profile.coverage -count=1 -timeout=30m ./...
-	cd mattermost-plugin/server; go tool cover -func plugin-postgres-profile.coverage
-	docker-compose -f ./docker-testing/docker-compose-postgres.yml down -v --remove-orphans
-
 webapp-ci: ## Webapp CI: linting & testing.
-	cd webapp; npm run check
-	cd mattermost-plugin/webapp; npm run lint
+	cd webapp; npm run check; npm run lint
```

Comment:
Does this also run the type checker on the web app code? That's might not be needed with Webpack using `ts-loader` though